contracts/container/contract.go

```diff
@@ -61,15 +61,16 @@ const (
 	// V2 format.
 	containerIDSize = interop.Hash256Len // SHA256 size
 
-	singleEstimatePrefix = "est"
-	estimateKeyPrefix    = "cnr"
-	containerKeyPrefix   = 'x'
-	ownerKeyPrefix       = 'o'
-	deletedKeyPrefix     = 'd'
-	nodesPrefix          = 'n'
-	replicasNumberPrefix = 'r'
-	nextEpochNodesPrefix = 'u'
-	estimatePostfixSize  = 10
+	singleEstimatePrefix     = "est"
+	estimateKeyPrefix        = "cnr"
+	containersWithMetaPrefix = 'm'
```

Comment:
just a remark

i very like (no) how gofmt makes a 10-line change when one is actually added/changed. Takin bout why linebreaks are good sometimes, and const/var blocks aint